yarn.lock

```diff
@@ -0,0 +1,901 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+"@ungap/promise-all-settled@1.1.2":
+  version "1.1.2"
+  resolved "https://npm.goodeggs.com/@ungap%2fpromise-all-settled/-/promise-all-settled-1.1.2.tgz#aa58042711d6e3275dd37dc597e5d31e8c290a44"
```

Comment:
@acbarnett Your build is failing because of these `npm.goodeggs.com` URLs that were added to the `yarn.lock` you added. This is because this public repo is missing an `.npmrc` file pointing at the public registry, so your machine used your user `.npmrc` which specifies our private registry as the default: https://goodeggs.atlassian.net/wiki/spaces/ENG/pages/459833453/Using+NPM+and+Yarn#I-ran-yarn-add-and-now-there-are-references-to-npm.goodeggs.com-in-yarn.lock-in-a-public-module!.

Reply:
Never mind, looks like you caught it right before!

Reply:
Actually, you should still add an `.npmrc` (see the doc).

Reply:
Thanks for pointing me to the docs! I'll add the `.npmrc` file with `registry = https://registry.npmjs.org/` right away.